client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HttpConnection.ts

```diff
@@ -32,18 +32,18 @@ export class HttpConnection implements IConnection {
         this.options = options;
     }
 
-    async start(): Promise<void> {
+    async start(requestedTransferMode: TransferMode): Promise<void> {
```

Comment:
What's the alternative? Features? Adding Features felt to heavy for this. We can do this but it would come separately.